knowhere/knowhere/archive/KnowhereConfig.cpp

```diff
@@ -8,19 +8,19 @@
 // Unless required by applicable law or agreed to in writing, software distributed under the License
 // is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 // or implied. See the License for the specific language governing permissions and limitations under the License.
-#ifdef __APPLE__
```

Comment:
I suggest to change CMakeLists.txt to let KnowhereConfig.cpp not compiled under Mac OS, instead of changing every APIs in this file.